Source/Core/AudioCommon/WaveFile.cpp

```diff
@@ -117,10 +117,10 @@ void WaveFileWriter::Write4(const char* ptr)
 void WaveFileWriter::AddStereoSamplesBE(const short* sample_data, u32 count, int sample_rate)
 {
   if (!file)
-    PanicAlertT("WaveFileWriter - file not open.");
+    PanicAlertFmtT("WaveFileWriter - file not open.");
```

Comment:
It feels a bit weird that this one is translated but the other one is not. Both messages seem like they don't make too much sense being either translated or shown to the user (since they likely can't do much about it anyways)